xbmc/guilib/GUIImage.cpp

```diff
@@ -334,40 +340,40 @@ void CGUIImage::SetFileName(const std::string& strFileName, bool setConstant, co
   { // texture is changing - attempt to load it, and save the name in m_currentTexture.
     // we'll check whether it loaded or not in Render()
     m_currentTexture = strFileName;
-    if (m_texture.SetFileName(m_currentTexture))
+    if (m_texture->SetFileName(m_currentTexture))
       MarkDirtyRegion();
   }
 }
 
 #ifdef _DEBUG
 void CGUIImage::DumpTextureUse()
 {
-  if (m_texture.IsAllocated())
+  if (m_texture->IsAllocated())
   {
     if (GetID())
-      CLog::Log(LOGDEBUG, "Image control %u using texture %s",
-                GetID(), m_texture.GetFileName().c_str());
+      CLog::Log(LOGDEBUG, "Image control %u using texture %s", GetID(),
+                m_texture->GetFileName().c_str());
     else
-      CLog::Log(LOGDEBUG, "Using texture %s", m_texture.GetFileName().c_str());
+      CLog::Log(LOGDEBUG, "Using texture %s", m_texture->GetFileName().c_str());
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)